README.md

```diff
@@ -117,6 +110,14 @@ We support deployment on GCP and Azure via Helm charts which takes 30-45mins. Yo
 
 You can experience the product by signing up for our [hosted sandbox](https://app.hyperswitch.io/). The signup process accepts any email ID and provides access to the entire Control Center. You can set up connectors, define workflows for routing and retries, and even try payments from the dashboard.
 
+<a href="#architectural-overview">
+  <h2 id="architectural-overview">Architectural Overview</h2>
```

Comment:
Can you update the table of contents to reflect this change in order of sections?